modules/contact-form/js/gutenblock.jsx

```diff
@@ -0,0 +1,568 @@
+					<GrunionFieldRequiredToggle
+						required={ this.props.required }
+						onChange={ ( x )=>this.props.setAttributes( { required: x } ) }
+					/>
+				</PanelBody>
+			</InspectorControls>
+		);
+	}
+}
+
+class GrunionFieldLabel extends Component {
+	render() {
+		return (
+			<Fragment>
+				<input
+					type='text'
+					value={ this.props.label }
+					className='grunion-field-label'
+					onChange={ ( x )=>this.props.setAttributes( { label: x.target.value } ) }
+				/>
+				{ this.props.required && <span className="required">{ __( '(required)' ) }</span> }
+			</Fragment>
+		);
+	}
+}
+
+class GrunionField extends Component {
+	render() {
+		return (
+			<Fragment>
+				<GrunionFieldSettings
+					required={ this.props.required }
+					setAttributes={ this.props.setAttributes }
+				/>
+				<div className="grunion-field">
+					<TextControl
+						type={ this.props.type }
+						label={ <GrunionFieldLabel
+							required={ this.props.required }
+							label={ this.props.label }
+							setAttributes={ this.props.setAttributes }
+						/> }
+						disabled={ true }
+					/>
+				</div>
+			</Fragment>
+		);
+	}
+}
+
+class GrunionFieldTextarea extends Component {
+	render() {
+		return (
+			<Fragment>
+				<GrunionFieldSettings
+					required={ this.props.required }
+					setAttributes={ this.props.setAttributes }
+				/>
+				<div className="grunion-field">
+					<TextareaControl
+						label={ <GrunionFieldLabel
+							required={ this.props.required }
+							label={ this.props.label }
+							setAttributes={ this.props.setAttributes }
+						/> }
+						disabled={ true }
+					/>
+				</div>
+			</Fragment>
+		);
+	}
+}
+
+class GrunionFieldCheckbox extends Component {
+	render() {
+		return (
+			<Fragment>
+				<GrunionFieldSettings
+					required={ this.props.required }
+					setAttributes={ this.props.setAttributes }
+				/>
+				<div className="grunion-field">
+					<CheckboxControl
+						label={ <GrunionFieldLabel
+							required={ this.props.required }
+							label={ this.props.label }
+							setAttributes={ this.props.setAttributes }
+						/> }
+						disabled={ true }
+					/>
+				</div>
+			</Fragment>
+		);
+	}
+}
+
+class GrunionFieldMultiple extends Component {
+	render() {
+		return (
+			<Fragment>
+				<GrunionFieldSettings
+					required={ this.props.required }
+					setAttributes={ this.props.setAttributes }
+				/>
+				<div className="grunion-field">
+					<GrunionFieldLabel
+						required={ this.props.required }
+						label={ this.props.label }
+						setAttributes={ this.props.setAttributes }
+					/>
+					<ol>
+						{ _.map( this.props.options, ( option, key, list )=>(
+							<li key={ key }>
+								<input
+									type="text"
+									className="option"
+									value={ option }
+									placeholder={ __( 'Enter your option value here…' ) }
+									onChange={ function( x ) {
+										const $options = jQuery( x.target ).closest( 'ol' ).find( 'input.option' );
```

Comment:
jQuery and React aren't terribly compatible. I'd try to do this without jQuery. Let me know if you'd like to discuss for more options so I can better understand what you're trying to accomplish here.